src/orion/client/experiment.py

```diff
@@ -383,20 +447,55 @@ def suggest(self):
             Reserved trial for execution. Will return None if experiment is done or broken
```

Comment:
```suggestion
            Reserved trial for execution. Will return None if experiment is done.
```
And remove rest of sentence.